cache/converter.go

```diff
@@ -132,15 +133,14 @@ func (c *conversion) convert(ctx context.Context, cs content.Store, desc ocispec
 	if err := w.Truncate(0); err != nil { // Old written data possibly remains
 		return nil, err
 	}
-	var zw io.WriteCloser = w
-	var compress io.WriteCloser
+	bufW := bufio.NewWriterSize(w, 128*1024)
```

Comment:
could we use a pool for this. Probably not very meaningful but I think it's a good practice to do it for bigger buffers.

Reply:
thanks for the comment, fixed.